utils/src/main/kotlin/Extensions.kt

```diff
@@ -257,6 +257,11 @@ fun String.fileSystemEncode() =
         .replace(Regex("(^\\.|\\.$)"), "%2E")
         .take(255)
 
+/**
+ * Return true is the string represents a true value, otherwise return false.
```

Comment:
Typo: **if**
Maybe be more specific and say that the string has to be "true" case-insensitive?

Reply:
> Typo: **if**

Fixed.

> Maybe be more specific and say that the string has to be "true" case-insensitive?

I didn't want to document the implementation details of `toBoolean()` / `parseBoolean()`, also to leave it open to in the future maybe extend this to also accept "yes", "1", and possible more as true values.